client/src/main/java/io/pravega/client/admin/impl/StreamManagerImpl.java

```diff
@@ -155,6 +156,12 @@ public boolean checkScopeExists(String scopeName) {
         return  Futures.getThrowingException(controller.checkScopeExists(scopeName));
     }
 
+    @Override
+    public List<TransactionInfo> listCompletedTransactions(Stream stream) {
+        log.info("Listing completed transaction for stream : {}", stream.getStreamName());
```

Comment:
nit: it should be "transactions" -- in plural

Reply:
done